docs/guidance/prometheus-grafana.md

```diff
@@ -0,0 +1,197 @@
+# Ray Cluster: Monitoring with Prometheus & Grafana
+
+This section will describe how to monitor Ray Clusters in Kubernetes using Prometheus & Grafana.
+
+If you do not have any experience with Prometheus and Grafana on Kubernetes, I strongly recommend you watch this [YouTube playlist](https://youtube.com/playlist?list=PLy7NrYWoggjxCF3av5JKwyG7FFF9eLeL4).
+
+## Step 1: Create a Kubernetes cluster with Kind.
+
+```sh
+kind create cluster
+```
+
+## Step 2: Install Kubernetes Prometheus Stack via Helm chart
+
+```sh
+# Path: kuberay/
+./install/prometheus/install.sh
+
+# Check the installation
+kubectl get all -n prometheus-system
+
+# (part of the output)
+# NAME                                                  READY   UP-TO-DATE   AVAILABLE   AGE
+# deployment.apps/prometheus-grafana                    1/1     1            1           46s
+# deployment.apps/prometheus-kube-prometheus-operator   1/1     1            1           46s
+# deployment.apps/prometheus-kube-state-metrics         1/1     1            1           46s
+```
+* KubeRay provides an [install.sh script](../../install/prometheus/install.sh) to install the [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) chart and related custom resources, including **ServiceMonitor** and **PodMonitor**, in the namespace `prometheus-system` automatically.
+
+## Step 3: Install a KubeRay operator
+
+```sh
+helm repo add kuberay https://ray-project.github.io/kuberay-helm/
```

Comment:
instead of duplicating this and potentially becoming out of date, should we just point to `helm-chart/kuberay-operator/README.md`?

same with ray cluster below

Reply:
Good point. Updated https://github.com/ray-project/kuberay/pull/895/commits/4bc3a748c31981cbc0659c53614546f4758e588a.